site/content/docs/07-state-management.md

```diff
@@ -13,22 +13,25 @@ Inside a component, get references to the stores like so:
 </script>
 ```
 
-* `preloading` contains a readonly boolean value, indicating whether or not a navigation is pending
-* `page` contains a readonly `{ host, path, params, query, error }` object. `error` is only set on error pages. This is the same value that is passed to `preload` functions.
-* `session` contains whatever data was seeded on the server. It is a [writable store](https://svelte.dev/tutorial/writable-stores), meaning you can update it with new data (for example, after the user logs in) and your app will be refreshed
+* `preloading` contains a read-only boolean value, indicating whether or not a navigation is pending
+* `page` contains information on the current address. See [preloading](docs#Arguments) for details. This store is read-only.
+* `session` can be used to pass data from the server related to the current request. It is a [writable store](https://svelte.dev/tutorial/writable-stores), meaning you can update it with new data. If, for example, you populate the session with the current user on the server, you can update the store when the user logs in. Your components will refresh to reflect the new state
 
 
 ### Seeding session data
 
-On the server, you can populate `session` by passing an option to `sapper.middleware`:
+`session` is `undefined` by default. To populate it with data, implement a function that returns session data given an HTTP request. Pass it as an option to `sapper.middleware` when the server is started.
+
+As an example, here is how to populate the session with the current user:
 
 ```js
 // src/server.js
-express() // or Polka, or a similar framework
```

Comment:
I would probably leave this comment for now so that people don't get confused by this saying `express` when the template uses `polka`. or maybe change `express` to `polka` here to match the template

Reply:
Oh. I was certainly confused by it. I thought this comment was part of the template code and wasn't relevant to the documentation. I'll switch to polka to match the sample then.